### DISCUSSION TITLE
How can I use docker on my synology diskstation with ytmdl-web-v2?

### DISCUSSION THREAD

**Discussion by appleinthefuture** [deepjyoti30/ytmdl-web-v2#58] [General]
I am new to github and docker. 
I would like to run ytmdl-web-v2 on my local diskstation.
Where can I find the image file, and how to deploy in docker?

Thank you

**Comment by deepjyoti30** [comment]
Hey @appleinthefuture as the README states, you can deploy locally using a command [as shown here](https://github.com/deepjyoti30/ytmdl-web-v2#deploy-locally).

You will just need to sign into `ghcr` through docker before you run the above command. [Read more on that here](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
